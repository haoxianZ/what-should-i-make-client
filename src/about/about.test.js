import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import About from './about';

describe(`About component`, () => {
  it('renders without crash', () => {
    const wrapper = shallow(<About />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
}) 