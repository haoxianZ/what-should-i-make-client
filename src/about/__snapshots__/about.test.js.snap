// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders without crash 1`] = `
<div>
  <Link
    style={
      Object {
        "textDecoration": "none",
      }
    }
    to="/home-page"
  >
    <header>
      What Should I Make?
    </header>
  </Link>
  <div
    className="about"
  >
    <h2>
      Welcome!
    </h2>
    <h3>
      What is this?
    </h3>
    <h4>
      This is an app to help you to decide what to cook base on what is in your fridge.
    </h4>
    <h3>
      How do I use this App?
    </h3>
    <h4>
      Create an account and log in what you have in the Fridge. Then explore what you can cook with them!
    </h4>
  </div>
</div>
`;
